src/common.ts

```diff
@@ -90,6 +90,16 @@ export async function getUnixChildProcessIds(pid: number): Promise<number[]> {
         });
 
         ps.on('error', reject);
+
+        ps.stderr.setEncoding('utf8');
```

Comment:
I don't know how this works with TS promises, but do you need to do something about the `ps.on('error', reject);` on L92 as well?

Reply:
I don't think it will be a problem. Looking at [the way vscode-node-debug](https://github.com/microsoft/vscode-node-debug/commit/52989207b9677da89d1f6b6a3852751787fdf2f7) handled error, it is effectively the same